build_test.go

```diff
@@ -0,0 +1,62 @@
+package enable
```

Comment:
Is this whole file actually needed? In the end, we just log something in the build part.

Reply:
Not really just for logging purposes, but I will remove it so it is cleaner

Reply:
removed